modules/apps/transfer/keeper/msg_server.go

```diff
@@ -50,10 +50,6 @@ func (k Keeper) Transfer(goCtx context.Context, msg *types.MsgTransfer) (*types.
 			sdk.NewAttribute(types.AttributeKeyDenom, msg.Token.Denom),
 			sdk.NewAttribute(types.AttributeKeyMemo, msg.Memo),
 		),
-		sdk.NewEvent(
```

Comment:
With the recent SDK changes, you need to re-add that.

Reply:
Yup, missed this when pulling in the latest sdk, reverting the changes atm. Thanks for the getting the updates to baseapp in so quick 🧙